praw/models/reddit/mixins/__init__.py

```diff
@@ -112,6 +112,42 @@ def remove(self, spam=False):
         data = {'id': self.thing.fullname, 'spam': bool(spam)}
         self.thing._reddit.post(API_PATH['remove'], data=data)
 
+    def send_removal_message(self, type,  # pylint: disable=redefined-builtin
+                             title, message):
+        """Send a removal message for a Comment or Submission.
+
+        Reddit adds human-readable information about the object to the message.
+
+        :param type: One of 'public', 'private', 'private_exposed'.
+            'public' leaves a stickied comment on the post.
+            'private' sends a Modmail message with hidden username.
+            'private_exposed' sends a Modmail message without hidden username.
+        :param title: The short reason given in the message.
+            (Ignored if type is 'public'.)
+        :param message: The body of the message.
+
+        If ``type`` is 'public', the new :class:`~.Comment` is returned.
+        """
+        # The API endpoint used to send removal messages is different
+        # for posts and comments, so the derived classes specify which one.
+        if self.REMOVAL_MESSAGE_API is None:
+            raise NotImplementedError('ThingModerationMixin must be extended.')  # NOQA
```

Comment:
It also looks like this is [the one line that isn't covered in tests](https://coveralls.io/builds/21033596/source?filename=praw%2Fmodels%2Freddit%2Fmixins%2F__init__.py#L134). A unit test like [this one](https://github.com/praw-dev/praw/blob/394ac595b2ea099bd31bb514233bdf30693cdb1f/tests/unit/models/list/test_base.py#L25-L27) would do the trick!